USAGE.md

```diff
@@ -69,6 +72,10 @@ Packwerk reads from the `packwerk.yml` configuration file in the root directory.
 | custom_associations  | N/A                                       | list of custom associations, if any |
 | parallel             | true                                      | when true, fork code parsing out to subprocesses |
 
+## Setting up Spring
+
+[Spring](https://github.com/rails/spring) is a preloader for Rails. Because `packwerk` loads `Rails`, it can be sped up dramatically by enabling spring.  Packwerk supports the usage of Spring. To enable Spring, first run `bin/spring binstub packwerk` which will "springify" the generated binstub. Secondly, spring needs to know about the packwerk spring command when spring is loading. To do that, add `require 'packwerk/spring_command'` to the bottom of `config/spring.rb` in your application.
```

Comment:
I my testing, it looks like I need to add the require to the spring config first, before I generate the binstub:

```
 $ bin/spring binstub packwerk
The 'packwerk' command is not known to spring.
```

Also, is there a reason we should recommend to add it to the _bottom_ of the file? Why not just say ```to `config/spring.rb` in your application.```?

Reply:
Noted -- I think I didn't properly capture this order dependency. Let me clarify to require and THEN run `bin/spring binstub packwerk`.

No -- I don't know of any reason to add it to the bottom. Let me remove that requirement!